client/components/Models.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { fetchProducts } from '../store/redux/products';
+import Model from './Model';
+
+class Models extends React.Component {
+  componentDidMount() {
+    this.props.getProducts();
+  }
+
+  render() {
+    console.log(this.props);
```

Comment:
Keep console.logs out of production code where you can. This can lead to security vulnerabilities